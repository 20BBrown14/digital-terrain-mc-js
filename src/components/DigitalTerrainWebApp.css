.web-view-layout-header.ant-layout-header {
  background: white;
  border-bottom: 1px solid #f0f0f0;
  height: 66px;
}
