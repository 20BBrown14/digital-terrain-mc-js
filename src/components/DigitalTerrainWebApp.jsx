import React from 'react';
import logo from '../logo.svg';
import './DigitalTerrainWebApp.css';

function DigitalTerrainWebApp() {
  return (
    <div className="digital-terrain-web-app">
      <header className="digital-terrain-web-app-header">
        <img src={logo} className="digital-terrain-web-app-logo" alt="logo" />
        <p>
          Edit <code>src/components/DigitalTerrainWebbApp.jsx</code> and save to reload yo.
        </p>
        <a
          className="digital-terrain-web-app-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default DigitalTerrainWebApp;
