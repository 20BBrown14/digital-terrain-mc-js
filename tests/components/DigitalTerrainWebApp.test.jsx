import React from 'react';
import DigitalTerrainWebApp from '../../src/components/DigitalTerrainWebApp';

describe('DigitalTerrainWebApp', () => {
  it('should render a default view', () => {
    const testApp = render(
      <DigitalTerrainWebApp />,
    );
    expect(testApp).toMatchSnapshot();
  });
});
