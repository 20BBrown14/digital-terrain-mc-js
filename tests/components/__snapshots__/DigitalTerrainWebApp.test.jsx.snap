// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DigitalTerrainWebApp should render a default view 1`] = `
<div
  class="digital-terrain-web-app"
>
  <header
    class="digital-terrain-web-app-header"
  >
    <img
      alt="logo"
      class="digital-terrain-web-app-logo"
      src="test-file-stub"
    />
    <p>
      Edit 
      <code>
        src/components/DigitalTerrainWebbApp.jsx
      </code>
       and save to reload yo.
    </p>
    <a
      class="digital-terrain-web-app-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
